.gitignore

```diff
@@ -75,6 +75,7 @@ tmp/
 outputs/
 multirun/
 mlruns/
+xgboost_gridsearch
```

Comment:
we might just want to move gridsearch results to a directory we ignore altogether?